roles/init_dbserver/defaults/main.yml

```diff
@@ -5,11 +5,19 @@ pg_version: 14
 pg_ssl: true
 pg_instance_name: main
 pg_remote_ssl_src: false
+
+# This parameter define if we want to run pg_initdb command on a server or not
+run_initdb: true
```

Comment:
I think this should be an inventory variable passed through the `pgd` dictionary or pgd role dependent instead of a flag.

Reply:
I thought about that. However, it will create challenges for PSR (Physical Streaming Replication).

Reply:
As discussed. Resolving this.